src/main/java/com/bulkbuy/enterprise/service/OrderServiceStub.java

```diff
@@ -1,4 +1,52 @@
 package com.bulkbuy.enterprise.service;
 
-public class OrderServiceStub {
+import com.bulkbuy.enterprise.dto.Order;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class that handles business logic for Order DTOs
+ */
+@Service
+public class OrderServiceStub implements IOrderService {
```

Comment:
While still working on the DAO's the business logic should be placed in a stub as the data is currently all hard coded